src/scripts/index.es6

```diff
@@ -1,16 +1,18 @@
-var jQueryFallbackProvider = require('./utils/jQueryFallbackProvider')
-var componentsHandler = require('./componentsHandler')
+require('./utils/swRegister')
 
+let jQueryFallbackProvider = require('./utils/jQueryFallbackProvider')
```

Comment:
:+1: Bylo by to lepší. Ten kód vznikal v době, kdy `mango-cli-example` bylo plné ještě `var`. A tomu tak stále je. :(